MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -29,7 +29,7 @@ DEPENDENCIES:
   - MapboxNavigation (from `../../../`)
 
 SPEC REPOS:
-  trunk:
+    https://github.com/cocoapods/specs.git:
```

Comment:
Yeah, this has been a longstanding annoyance for us. To keep the `trunk` repo from being a problem in the future, we’d need to ensure that the CI bots are all running a newer version of CocoaPods that can handle `trunk`. Some of the older-Xcode CI bots are on images that come with older CocoaPods versions. We could either manually upgrade CocoaPods on those bots as part of the build process, or we could upgrade those bots to a newer-Xcode image.